.gitlab/devcontainer.yml

```diff
@@ -16,7 +16,7 @@ lint_devcontainer:
     matrix:
       - PLATFORM: [ amd64,arm64 ]
   script:
-    - docker buildx build --no-cache --platform linux/${PLATFORM} --tag ${DOCKER_TARGET}-${PLATFORM} -f devcontainer/Dockerfile . ${PUSH}
+    - docker buildx build --build-arg GO_VERSION=$(grep GO_VERSION go.env | cut -d = -f 2) --no-cache --platform linux/${PLATFORM} --tag ${DOCKER_TARGET}-${PLATFORM} -f devcontainer/Dockerfile . ${PUSH}
```

Comment:
Could/should we do exactly [like the others](https://github.com/DataDog/datadog-agent-buildimages/blob/main/.gitlab-ci.yml#L116-L117)?

Reply:
I switched to this way but kept the grep because I don't need the other variables! Thanks